docs/datastore/cli.md

```diff
@@ -0,0 +1,38 @@
+---
+id: cli
+title: Datastore CLI
+sidebar_label: Datastore CLI
+---
+
+## Using the CLI tool
+
+We provide a graphback plugin to generate required config for datastore.
```

Comment:
If I put myself as newbie I will really struggle to get sense from this. 
There is strong assumption that someone will have graphback installed. 

There is no single cli or instruction to get that done. 
Not sure if that is problem with doc, but flagging this as docs here are really confusing and we simply missing info in order to replicate this from beginning.

Reply:
I haven't updated this

Reply:
We don't have an offix cli. When we discussed clis, you said it would be difficult to manage two clis. I don't know how I'm going to spin this. They have to install graphback. I already point to graphback docs. I will add small Graphback intro but I don't think it will make much difference since they still have to go to the graphback docs.

Reply:
I have commented that while ago. We kinda need small wrapper around graphback that will create config in memory rather than in the project. This will simplify docs.

Reply:
Does the Graphback cli support such customization?

Reply:
Yep. It is programatical usage vs cli/config (we support both)
Docs shows us missing pieces so.

https://github.com/aerogear/offix/issues/729